src/GitVersionTfsTask/Extension/vss-extension.json

```diff
@@ -1,9 +1,8 @@
 {
     "manifestVersion": 1,
-    "id": "gitversion",
+    "id": "gl-gitversion",
```

Comment:
I'll revert it when it's ready, meanwhile it allows me to publish a prerelease version of it without any conflicts :)